fixtures/american-dj/vizi-q-wash7.json

```diff
@@ -0,0 +1,314 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Vizi Q Wash7",
+  "shortName": "Q7",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Rey Victor"],
+    "createDate": "2022-12-17",
+    "lastModifyDate": "2022-12-17"
+  },
+  "comment": "The Vizi Q Wash7 is a 280W professional moving head wash fixture with variable zoom (5 ~ 55-degrees) and seven 40W Quad (RGBW) LEDs. At 280W, this fixture offers very powerful 5-degree beams or brilliant 55-degree washes of color. WiFLY wireless DMX on-board.",
+  "links": {
+    "manual": [
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/10532/ADJ%20VIZI%20Q%20WASH7%20-%20USER%20MANUAL%20.pdf"
+    ],
+    "productPage": [
+      "https://www.adj.com/vizi-q-wash7"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=EHeTZ_G_8zM"
+    ]
+  },
+  "physical": {
+    "dimensions": [284.4, 411.7, 202],
+    "weight": 8.5,
+    "power": 280,
+    "DMXconnector": "3-pin and 5-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [7, 58]
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "defaultValue": "50%",
+      "capability": {
+        "type": "Pan",
+        "angleStart": "0deg",
+        "angleEnd": "630deg"
+      }
+    },
+    "Tilt": {
+      "defaultValue": "50%",
+      "capability": {
+        "type": "Tilt",
+        "angleStart": "-160deg",
+        "angleEnd": "160deg"
+      }
+    },
+    "Red": {
+      "defaultValue": "0%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "defaultValue": "0%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "defaultValue": "0%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "defaultValue": "100%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Color Macros": {
+      "defaultValue": "0%",
+      "capability": {
+        "helpWanted": "What are the Color Macros?",
```

Comment:
These may well be in the manual, I'll come back to this later.

Reply:
They are, on page 20. Do you have the time to fill them in?

Reply:
Yes, I've been super busy in the run up to Christmas but I can pass a few hours over the holidays blasting through some open PR's.

Reply:
@FloEdelmann As the fixture is RGBW, do you have a tried-and-tested way of converting the RGBW values to RGB hex?

Reply:
I wonder if it shares the same colour presets as the [Saber Spot RGBW](https://github.com/OpenLightingProject/open-fixture-library/blob/5f44c0f3db5dbea180c335f95041ba3af52291c7/fixtures/american-dj/saber-spot-rgbw.json) since it is the same manufacturer? At a glance, they do look similar.

Reply:
Indeed, they look the same. Good that you saved the manual work by copying!

Unfortunately, there is no tried and true method. See also:

* #437